### DISCUSSION TITLE
Organic Map crash

### DISCUSSION THREAD

**Discussion by MXC48-zz** [organicmaps/organicmaps#2158] [Q&A]
Hi, it's been a few updates that the application crashes during startup
Can anyone explain this?

**Comment by biodranik** [comment]
Can you please clarify your situation? Is it iOS or Android? When exactly does it crash? Does it work later? On Android it is possible to connect the device via USB cable and record/share crash logs using `adb logcat` command, so we can take a look.

**Reply by MXC48-zz** [reply]
Thank you but I reinstalled the application and it worked. So I don't know what happened

**Reply by biodranik** [reply]
Do you have Android with SD card? It could be a corrupted SD.
